widgetry/src/backend_glow.rs

```diff
@@ -472,6 +475,28 @@ impl PrerenderInnards {
             self.gl.use_program(Some(self.program));
         }
     }
+
+    pub fn upload_texture(&self, texture: SpriteTexture, scale: (f32, f32)) {
+        if self.is_gl2 {
+            texture
+                .upload_gl2(&self.gl)
+                .expect("failed to upload textures");
+
+            unsafe {
```

Comment:
It doesn't seem useful to set this uniform every single time we draw; it only changes when we set the texture